src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -287,42 +287,44 @@ public with sharing class CRLP_RollupUI_SVC {
     */
 
     @AuraEnabled
-    public static FilterGroupModel setupFilterGroupDetail(Id id, List<String> objectNames) {
-
+    public static FilterGroupModel setupFilterGroupDetail(String filterGroupId, List<String> objectNames) {
```

Comment:
@randi274 It's going to be a major challenge to merge this because nearly all of the code in this method has been re-written to not use `FilterRulesTableItem`. In fact, I've removed `FilterRulesTableItem` entirely from the class because for the save logic to work the objects passed back to the component are the `FilterGroup` and `FilterRule` inner classes from `CRLP_RollupCMT`. It's the same logic used for saving Rollups. I hate to scrap this PR, but I'm not sure how we're going to reconcile the changes. Let's talk about this tomorrow.